spark/v3.2/spark/src/main/java/org/apache/iceberg/spark/source/SparkWrite.java

```diff
@@ -622,6 +625,7 @@ public DataWriter<InternalRow> createWriter(int partitionId, long taskId, long e
       FileIO io = table.io();
 
       OutputFileFactory fileFactory = OutputFileFactory.builderFor(table, partitionId, taskId)
+          .operationId(queryId)
```

Comment:
[question] should we also do it for [SparkPositionDeltaWrite](https://github.com/apache/iceberg/blob/master/spark/v3.2/spark/src/main/java/org/apache/iceberg/spark/source/SparkPositionDeltaWrite.java#L321-L326) used in mor.